use-cases/rag-pipeline/alloy-db-setup/src/alloy-db-setup-job.py

```diff
@@ -49,33 +46,34 @@
 if __name__ == "__main__":
     # Configure logging
     logging.config.fileConfig("logging.conf")
-
-    logger = logging.getLogger("alloydb-catalog-onboarding")
+    logger = logging.getLogger(__name__)
```

Comment:
This will pickup `__main__` which would inherit the `root` logger. We might want to explicitly set this to `alloy-db` or `alloy-db-setup-job`

Reply:
This actually picks the name of file it is running from.

https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial

```A good convention to use when naming loggers is to use a module-level logger, in each module which uses logging, named as follows:

logger = logging.getLogger(__name__)
This means that logger names track the package/module hierarchy, and it’s intuitively obvious where events are logged just from the logger name.```

Reply:
Yes, but in this instance it picks up `__main__` because it is a script and not a module. You do not have an entry for that in your logging configuration so it will fall back to the root logger configuration.